apps/src/applab/designProperties.jsx

```diff
@@ -47,16 +59,113 @@ var DesignProperties = module.exports = React.createClass({
         handleDelete={this.props.onDelete}/>);
     }
 
+    var tabHeight = 35;
+    var borderColor = '#c6cacd';
+    var bgColor = '#e7e8ea';
+
+    var defaultTabStyle = {
+      borderColor: borderColor,
+      borderStyle: 'solid',
+      borderWidth: '1px 1px 1px 0',
+      boxSizing: 'border-box',
+      height: tabHeight,
+      padding: '0 10px'
+    };
+    var activeTabStyle = {
+      backgroundColor: bgColor,
+      borderWidth: '1px 1px 0 0'
+    };
+    var propertiesTabStyle = $.extend({}, defaultTabStyle, {
+      float: 'left'
+    }, this.state.selectedTab === TabType.PROPERTIES ? activeTabStyle : {});
+    var eventsTabStyle = $.extend({}, defaultTabStyle, {
+      float: 'left'
+    }, this.state.selectedTab === TabType.EVENTS ? activeTabStyle : {});
```

Comment:
On a related note, I wonder if it would be to have most of this style stuff live outside of the class, and only have stuff that changes based on state live inside render. This would make it more clear what is static vs dynamic.